src/pybind/kaldi_pybind.cc

```diff
@@ -29,8 +27,8 @@ namespace py = pybind11;
 using namespace kaldi;
 
 template<class Holder>
-void sequential_matrix_reader(py::module &m, std::string suffix) {
-    std::string name = "SequentialBaseFloatMatrixReader" + suffix;
+void sequential_matrix_reader(py::module &m, std::string data_type) {
+    std::string name = "SequentialBaseFloat" + data_type + "Reader";
     py::class_<SequentialTableReader<Holder > >(m, name.c_str())
```

Comment:
if you're using c++11, there is no need to place spaces between ">>". That is,
`py::class_<SequentialTableReader<Holder>>` is fine and will not cause any problems.
In addition, it is more compact.